public/sw.js

```diff
@@ -0,0 +1,51 @@
+const CACHE_VERSION = 'cache-version-1'
```

Comment:
Как-то очень длинно назвал.

```suggestion
const CACHE_VERSION = 'v1'
```